style.css

```diff
@@ -38,6 +38,24 @@ body {
     column-gap: 3px;
     margin-bottom: 0.1em;
     grid-template-columns: repeat(4, 1fr);
+    position: relative;
+}
+
+
+.calc-history {
+    position: absolute;
+    z-index: 9999;
+    width: 400px;
+    height: 70%;
+    background-color: var(--black);
+}
+
+.hidden {
+    display: none;
+}
+
+.calc-history p {
+    color: var(--white);
```

Comment:
Why not just add `color: var(--white)` on the `.calc-history` class above? It should have the same effect